### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:
1. Add style to the app.
2. Modify the structure to match the design.
3. Fix minor bugs.

### CODE REVIEW

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Lordkaito, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] The avatar is aligned vertically with list of books and the `ADD BOOK` button in the original design. It would be a good idea to follow the given design as close as possible. Please make the necessary changes.
- [x] The text of the button for adding a new book is `ADD BOOK` not `Add Books`. Please follow the given design.
- [x] The `Bookstore`, `BOOKS` and `CATEGORIES` links are not vertically aligned and the space between the `BOOKS` and `Bookstore` and the `BOOKS` and the `CATEGORIES` links are not uniform. Make the necessary changes.

## Original design
main-ui

## Your design
review


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 1]
```diff
@@ -1,8 +1,231 @@
...
+
+.linksUl a {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+}
```
Comment (omar-labana) [inline src/App.css]: - [x] The avatar icon is still not aligned properly, I'd suggest using `justify-between` to push the two-child div into the proper place.

File: src/components/AddBook.js [hunk 2]
```diff
@@ -21,29 +21,34 @@ const AddBook = () => {
...
+            id="booksCats">
+            <option disabled={true} value="default">Select a category</option>
+            <option value="Action">Action</option>
+            <option value="Manga">Manga</option>
+            <option value="Science Fiction">Science Fiction</option>
+            <option value="Heroes">Heroes</option>
+            <option value="Drama">Drama</option>
+            <option value="Horror">Horror</option>
+          </select>
+          <button className='addBook' type="submit">Add Book</button>
```
Comment (omar-labana) [inline src/components/AddBook.js]: As the previous reviewer highlighted, kindly change `Add Books` to ADD BOOK.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Lordkaito, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] _[OPTIONAL]_ Kindly update the README file with the correct instructions to get started(`yarn start` will probably create a conflict since you are using `npm`)
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-labana — APPROVED** [review]
Hi @Lordkaito, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] Kindly try using a `flex` container sling with `justify-items: between` for your header and 'ADD BOOK' form to align them easily.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**